ptx/sec_space_coord.ptx

```diff
@@ -1679,7 +1885,15 @@
               <caption/>
                 <!-- START figures/figsurfrev2b_3D.asy -->
               <image xml:id="img_surfrev2b_3D">
-                <description></description>
+                <shortdescription>
+                  Graph showing surface of revolution of function z= sin(x)  around x axis.
+                </shortdescription>
+                <description>
+                  <p>
+                    The surface of equation <m>z= \sin(\sqrt (x^2 +y^2)</m> is made by revolving 
```

Comment:
You will need braces for the square root argument: `\sqrt{x^2+y^2}`. And you are missing a closing parenthesis.